Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Spawnable/PrefabCatchmentProcessor.cpp

```diff
@@ -64,11 +64,9 @@ namespace AzToolsFramework::Prefab::PrefabConversionUtils
             AZStd::move(uniqueName), context.GetSourceUuid(), AZStd::move(serializer));
         AZ_Assert(spawnable, "Failed to create a new spawnable.");
 
-        Instance instance;
-        if (Prefab::PrefabDomUtils::LoadInstanceFromPrefabDom(
-                instance, prefab, object.GetReferencedAssets(),
-                Prefab::PrefabDomUtils::LoadFlags::AssignRandomEntityId)) // Always assign random entity ids because the spawnable is
-                                                                          // going to be used to create clones of the entities.
+        Instance instance(context.FindOrCreateCachedInstanceAliasForPrefab(prefabName));
+        if (Prefab::PrefabDomUtils::LoadInstanceFromPrefabDom(instance, prefab, object.GetReferencedAssets())) 
+                // Do not assign random entity ids here because other prefab processors may rely on consistent cross-spawnable entity IDs 
```

Comment:
I don't think storing the instance alias in the PrefabProcessorContext will cause the editor and game contexts to have the same entity ids. The editor entity ids come from the hashed value corresponding to the entity alias paths calculated from the root prefab instance, which is stored in PrefabEditorEntityOwnershipService. So, ctrl+g should still work. @SergeyAMZN , can you confirm that ctrl+g mode still works after this change?

Regarding this change in general, the entity alias being stored in this map here will provide an option for different prefab processors to generate same ids when loading the same dom. It can be used for usecases like this. For processors that always want to generated unique ids per instance, they can choose to simple do 'Instance instance()' instead of 'Instance instance(context.FindOrCreateCachedInstanceAliasForPrefab(prefabName))'. Since this change doesn't limit unique ids per instance, this change should be okay imo.

Reply:
Yes, that's correct, confirmed Ctrl+G works fine and the game time entity IDs are different from the Edit-time. 
In fact EditorInfoRemover still uses AssignRandomEntityId and just constructs an Instance without using the cached value